NetKAN/BDAnimationModules.netkan

```diff
@@ -1,11 +1,21 @@
 {
-    "spec_version": 1,
-    "identifier": "BDAnimationModules",
-    "$kref": "#/ckan/github/PapaJoesSoup/BDAnimationModules",
-    "name": "BahamutoD Animation Modules",
-    "abstract": "Animation modules plugin for KSP.",
-    "author": [ "BahamutoD", "Papa_Joe" ],
-    "license": "CC-BY-SA-2.0",
-    "ksp_version": "1.1.3",
-    "x_netkan_epoch": 1
+    "spec_version" : "v1.4",
+    "identifier"   : "BDAnimationModules",
+    "$kref"        : "#/ckan/github/raidernick/BDAnimationModules",
+    "name"         : "BahamutoD Animation Modules",
+    "abstract"     : "Animation modules plugin for KSP",
+    "comment"      : "I have taken to updating the mod again instead of Papa Joe, made changes accordingly",
+    "license"      : "CC-BY-SA-2.0",
+    "ksp_version_min"  : "1.2",
+    "ksp_version_max"  : "1.2",
```

Comment:
Since you have a .version file included in the zip, you can replace these hard-coded KSP version numbers with a $vref line:

    "$vref": "#/ckan/ksp-avc",